frontend/android/app/src/main/java/com/dailydash/MainApplication.java

```diff
@@ -4,6 +4,8 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import cl.json.RNSharePackage;
```

Comment:
Could you remove these? They are causing codacy errors

Reply:
done